test/mocks/cams_mock.exs

```diff
@@ -0,0 +1,98 @@
+    headers: [{"Date", "Tue, 26 Dec 2017 12:16:11 GMT"}, {"Server", "Apache"},
+    {"Content-Length", "137"}, {"Content-Type", "text/html; charset=UTF-8"}],
+    request_url: "https://secure.centralams.com/gw/api/transact.php",
+    status_code: 200}}
+  end
+  def invalid_transaction_id do
+    {:ok,
+    %HTTPoison.Response{body: "response=3&responsetext=Transaction not found REFID:3503243979&authcode=&transactionid=3921118690&avsresponse=&cvvresponse=&orderid=&type=capture&response_code=300",
+    headers: [{"Date", "Tue, 26 Dec 2017 12:39:05 GMT"}, {"Server", "Apache"},
+    {"Content-Length", "163"}, {"Content-Type", "text/html; charset=UTF-8"}],
+    request_url: "https://secure.centralams.com/gw/api/transact.php",
+    status_code: 200}}   
+  end 
+  def more_than_authorization_amount do  
+    {:ok,
+    %HTTPoison.Response{body: "response=3&responsetext=The specified amount of 1001 exceeds the authorization amount of 1000.00 REFID:3503244462&authcode=&transactionid=3921127126&avsresponse=&cvvresponse=&orderid=&type=capture&response_code=300",
+    headers: [{"Date", "Tue, 26 Dec 2017 13:00:55 GMT"}, {"Server", "Apache"},
+    {"Content-Length", "214"}, {"Content-Type", "text/html; charset=UTF-8"}],
+    request_url: "https://secure.centralams.com/gw/api/transact.php",
+    status_code: 200}}
+  end
+  def successful_refund do
+    {:ok,
+    %HTTPoison.Response{body: "response=1&responsetext=SUCCESS&authcode=&transactionid=3921158933&avsresponse=&cvvresponse=&orderid=&type=refund&response_code=100",
+    headers: [{"Date", "Tue, 26 Dec 2017 14:00:08 GMT"}, {"Server", "Apache"},
+    {"Content-Length", "131"}, {"Content-Type", "text/html; charset=UTF-8"}],
+    request_url: "https://secure.centralams.com/gw/api/transact.php",
+    status_code: 200}}
+  end
+
+  def more_than_purchase_amount do
+    {:ok,
+    %HTTPoison.Response{body: "response=3&responsetext=Refund amount may not exceed the transaction balance REFID:3503249728&authcode=&transactionid=&avsresponse=&cvvresponse=&orderid=&type=refund&response_code=300",
+    headers: [{"Date", "Tue, 26 Dec 2017 14:05:31 GMT"}, {"Server", "Apache"},
+    {"Content-Length", "183"}, {"Content-Type", "text/html; charset=UTF-8"}],
+    request_url: "https://secure.centralams.com/gw/api/transact.php",
+    status_code: 200}}   
+  end 
+  def successful_void do
+    {:ok,
+    %HTTPoison.Response{body: "response=1&responsetext=Transaction Void Successful&authcode=123456&transactionid=3921178863&avsresponse=&cvvresponse=&orderid=&type=void&response_code=100",
+    headers: [{"Date", "Tue, 26 Dec 2017 14:26:05 GMT"}, {"Server", "Apache"},
+    {"Content-Length", "155"}, {"Content-Type", "text/html; charset=UTF-8"}],
+    request_url: "https://secure.centralams.com/gw/api/transact.php",
+    status_code: 200}} 
+  end
+ 
+end
+
+
+
+
```

Comment:
Indentation is not proper and so many blank lines have been added at the end of the file.